WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/CustomerSelectorViewController.swift

```diff
@@ -187,6 +244,12 @@ private extension CustomerSelectorViewController {
     }
 
     func onCustomerTapped(_ customer: Customer) {
+        // Show alert if selecting guest is disallowed
+        if viewModel.configuration.disallowSelectingGuest && customer.customerID == 0 {
```

Comment:
Suggestion: we can maybe add a helper computed variable for customer to check if they're a guest to avoid comparing the ID every time:

```Swift
var isGuest: Bool {
    customerID == 0
}
```

Reply:
Good idea, https://github.com/woocommerce/woocommerce-ios/pull/12100/commits/60ee8319e12f20bb4af6367844ce16e08c7e64fe